zone/perl_bot.cpp

```diff
@@ -342,7 +342,7 @@ bool Perl_Bot_ReloadBotDataBuckets(Bot* self)
 
 bool Perl_Bot_ReloadBotOwnerDataBuckets(Bot* self)
 {
-	return self->GetBotOwnerDataBuckets();
+	return self->GetBotOwner()->CastToClient()->GetDataBuckets();
```

Comment:
May be worth checking if `GetBotOwner()` is valid in case this fires on death or something weird.